src/d3a/models/area/stats.py

```diff
@@ -140,19 +140,18 @@ def current_market(self):
         return past_markets[-1] if len(past_markets) > 0 else None
 
     def get_market_stats(self, market_slot_list):
-        if self.current_market is None:
-            return {"INFO": "No market stats available yet"}
+        default_stats_dict = {"min_trade_rate": None,
+                              "max_trade_rate": None,
+                              "avg_trade_rate": None,
+                              "median_trade_rate": None,
+                              "total_traded_energy_kWh": None}
```

Comment:
Can you please make this a global variable and reuse it in the `min_max_avg_median_rate_current_market`?
Thanks!